### PR TITLE
Reinicio automatico

### PR DESCRIPTION
**Pull request by gbarogo**
Automatic restart using cron

### CODE REVIEW

**Comment by esanchma** [comment]
@marcosio in this PR we are using cron so the nodes self-reboot each 2 days, with no participation from 3rd parties. Does this make sense to you?

**Comment by marcosio** [comment]
Hi @esanchma !!!

Great job!!!  Absolutely in accordance with your approach!

The only one is that nano isn't used in any script.

**Comment by esanchma** [comment]
> The only one is that nano isn't used in any script.

Yes, let's remove that just before merging

File: scripts/start.sh [hunk 1]
```diff
@@ -125,7 +125,7 @@ if ( [ ! -e /etc/cron.d/restart-node-cron ] ); then
 	superuser chmod 0644 /etc/cron.d/restart-node-cron
 fi
 
-if (ps -e | grep cron >/dev/null ); then
+if !(ps -e | grep cron >/dev/null ); then
```
Comment (esanchma) [inline scripts/start.sh]: https://linux.die.net/man/1/pgrep ?

**Comment by marcosio** [comment]
This pull request is closed because it is deprecated.

**Pull request closed without merging**